ush/rocoto/workflow_utils.py

```diff
@@ -262,7 +262,7 @@ def get_gfs_interval(gfs_cyc):
     if gfs_cyc == 0:
         interval = None
     if gfs_cyc == 1:
-        interval = '24:00:00'
+        interval = '120:00:00'
```

Comment:
Please revert this change. "interval" will always be 24hrs for gfs_cyc=1. Users who wish to run only 00z and only every 5 days (interval=120) should change their xml's manually after creation right now. If this is a desired setup that a number of users will want to use (00z every 5 days) let me know and the workflow team can add in that functionality. Thanks!